lib/normalizeOptions.js

```diff
@@ -85,7 +85,17 @@ function normalizeOptions(loaderContext, content, webpackImporter) {
   options.importer = options.importer
     ? proxyCustomImporters(options.importer, resourcePath)
     : [];
-  options.importer.push(webpackImporter);
+
+  if (options.resolver === false) {
+    // disable webpack resolved in favor built-in node-sass/sass resolver
+  } else if (typeof options.resolver === 'function') {
+    options.importer = options.importer.concat(
+      proxyCustomImporters(options.resolver, resourcePath)
+    );
```

Comment:
I think better don't add `webpackImporter` here. We can export our importer and developers decide whether he needs him or not

Reply:
Okay, removed webpackImporter by default.